armi/reactor/assemblies.py

```diff
@@ -15,25 +15,26 @@
 """
 Assemblies are collections of Blocks.
 
-Generally, blocks are stacked from bottom to top.
+Generally, Blocks are stacked from bottom to top.
 """
 import copy
 import math
 import pickle
 from random import randint
+from typing import ClassVar, Type
```

Comment:
Adding to make the `Assembly._BLOCK_TYPE` hint complete
```suggestion
from typing import ClassVar, Optional, Type
```